### DISCUSSION TITLE
Info of Satellite layer.

### DISCUSSION THREAD

**Discussion by hoangdangcodecomplete** [tomik23/react-leaflet-examples#10] [Q&A]
Hi, my friend @tomik23,
I wanna add the layer Satellite on my map, but I can't get the URL and attribution of this layer.
So can you share this info about the layer?
The layer like that:  ![Screen Shot 2022-06-06 at 10 29 11](https://user-images.githubusercontent.com/104552361/172089829-5e9cb927-efe3-4792-82ea-ec5bf1fc3f3d.png)

**Comment by tomickigrzegorz** [comment]
OSM does not have a satellite layer. You have to search and add yourself. Just type in google "osm satellite".

**Reply by hoangdangcodecomplete** [reply]
Thank a lot my friend!
